index.html

```diff
@@ -0,0 +1,176 @@
+      <h3>Pricing</h3>
+      <div>
+        <p>1: $4.99/each</p>
+      </div>
+      <div>
+        <p>2-50: $3.99/each</p>
+      </div>
+      <div>
+        <p>51+: $2.99/each</p>
+      </div>
+    </section>
+  </article>
+
+  <article class="red-card">
+    <header class="red-header">
+      <h1>Red Dunk Cards</h1>
+    </header>
+    <section>
+      <div>
+        <img class="image"  src="red.png" alt="red dunk card">
+      </div>
+      <div>
+        <p class="description">Squid poutine lomo, church-key tofu wayfarers salvia whatever nostrud readymade est farm-to-table yr.</p>
+      </div>
+      <div>
+        <p class="avail">Available</p>
+      </div>
+    </section>
+    <section class="specs">
+      <h4>Specifications</h4>
+        <div>
+          <p>Size: 4" x 4"</p>
+        </div>
+        <div>
+          <p>Weight: 8.2 oz.</p>
+        </div>
+        <footer>
+          <p>Specifications valid thru 2016</p>
+        </footer>
+    </section>
+    <section class="price">
+      <h3>Pricing</h3>
+      <div>
+        <p>1: $4.99/each</p>
+      </div>
+      <div>
+        <p>2-50: $3.99/each</p>
+      </div>
+      <div>
+        <p>51+: $2.99/each</p>
+      </div>
+    </section>
+  </article>
+
+  <article class="blue-card">
+    <header class="blue-header">
+      <h1>Blue Dunk Cards</h1>
+    </header>
+    <section>
+      <div>
+        <img class="image"  src="blue.png" alt="blue dunk card">
+      </div>
+      <div>
+        <p class="description">Try-hard non viral yr aesthetic. Single-origin coffee photo booth exercitation, thundercats pug.</p>
+      </div>
+      <div>
+        <p class="avail">Available</p>
+      </div>
+    </section>
+    <section class="specs">
+      <h4>Specifications</h4>
+        <div>
+          <p>Size: 4" x 4"</p>
+        </div>
+        <div>
+          <p>Weight: 8.2 oz.</p>
+        </div>
+        <footer>
+          <p>Specifications valid thru 2016</p>
+        </footer>
+    </section>
+    <section class="price">
+      <h3>Pricing</h3>
+      <div>
+        <p>1: $4.99/each</p>
+      </div>
+      <div>
+        <p>2-50: $3.99/each</p>
+      </div>
+      <div>
+        <p>51+: $2.99/each</p>
+      </div>
+    </section>
+  </article>
+
+  <article  class="pink-card">
+    <header class="pink-header">
+      <h1>Pink Dunk Cards</h1>
+    </header>
+    <section>
+      <div>
+        <img class="image"  src="pink.png" alt="pink dunk card">
+      </div>
+      <div>
+        <p class="description">Lumbersexual flexitarian XOXO do. Ex magna hammock, esse consectetur sunt accusamus.</p>
+      </div>
+      <div>
+        <p class="avail">Available</p>
+      </div>
+    </section>
+    <section class="specs">
+      <h4>Specifications</h4>
+        <div>
+          <p>Size: 4" x 4"</p>
+        </div>
+        <div>
+          <p>Weight: 8.2 oz.</p>
+        </div>
+        <footer>
+          <p>Specifications valid thru 2016</p>
+        </footer>
```

Comment:
These divs and footer don't need to be indented this far, they are on the same level as the h4.
The <p> tags also count as block level, so they don't need to be wrapped in a div, but if you were doing it for css targeting reasons I definitely understand.
Good Job!